validator/api/src/main/java/tech/pegasys/teku/validator/api/KeyStoreFilesLocator.java

```diff
@@ -57,7 +58,9 @@ public KeyStoreFilesLocator(final List<String> colonSeparatedPairs, final String
       final List<String> entry = Splitter.on(pathSeparator).limit(2).splitToList(currentEntry);
       parseEntry(entry.get(0), entry.get(1), pathMap);
     }
-    return getFilePairs(pathMap);
+
+    List<Pair<Path, Path>> filePairs = getFilePairs(pathMap);
+    return filePairs != null ? filePairs : Collections.emptyList();
```

Comment:
still checking nulls, i thought we decided it couldn't return null...

 - if getFilePairs is package private and static, we write a pretty simple test in `KeyStoreFilesLocatorTest`
 ```
   @Test
  void shouldNotReturnNull() {
    assertThatThrownBy(() -> KeyStoreFilesLocator.getFilePairs(null)).isInstanceOf(NullPointerException.class);
    assertThat(KeyStoreFilesLocator.getFilePairs(Map.of())).isNotNull();
  }
  ```
 - shows if it's accessed incorrectly its an NPE
 - shows if its accessed and doesn't throw, it never produces a null result

This test doesn't need to stay, but it does prove the question. so above we don't need to check for null.